pkg/cmd/release/update_releases_test.go

```diff
@@ -39,10 +39,6 @@ func Test_processReleaseData(t *testing.T) {
 			Predecessor: "22.2",
 			Withdrawn:   []string{"23.1.0"},
 		},
-		"23.2": {
```

Comment:
to confirm, this removal/cleanup is more related to advancing `MinSupported` to 24.3 (than to creating 24.3 and 25.1 start version gates, right?)

(mostly to clarify for when we refer to this for the next release cycle/task)

Reply:
This is just a test for the updated releases code, which we updated to ignore beta builds. This test runs against fixed data, the versions should be considered fictitious.